package-lock.json

```diff
@@ -13301,7 +13301,10 @@
       "version": "1.3.1",
       "resolved": "https://registry.npmjs.org/full-icu/-/full-icu-1.3.1.tgz",
       "integrity": "sha512-VMtK//85QJomhk3cXOCksNwOYaw1KWnYTS37GYGgyf7A3ajdBoPGhaJuJWAH2S2kq8GZeXkdKn+3Mfmgy11cVw==",
-      "dev": true
+      "dev": true,
```

Comment:
is this whole file needed? we don't usually ship `package-lock` unless there are changes in `package.json` as well

Reply:
Sorry but this was a mistake, it shouldn't have been uploaded

Reply:
no problem, that's what the reviews are for 😸